modules/core/src/main/java/org/apache/ignite/internal/processors/cache/ExchangeContext.java

```diff
@@ -37,7 +38,8 @@
  *
  */
 public class ExchangeContext {
-    /** */
+    /** Enables exchange compatibility with protocol version 1. */
+    @SystemProperty(value = "Enables exchange compatibility with protocol version 1.")
```

Comment:
I would say "Enables the compatibility mode for the exchange protocol of version 1."

Reply:
Fixed.